modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/components/SimpleInputModal.es.js

```diff
@@ -161,9 +175,21 @@ const SimpleInputModal = ({
 								disabled={loadingResponse}
 								id={`${namespace}${mainFieldName}`}
 								name={`${namespace}${mainFieldName}`}
-								onChange={(event) =>
-									setInputValue(event.target.value)
-								}
+								onChange={(event) => {
+									if (errorMessage) {
+										setErrorMessage('');
+									}
+
+									if (!event.target.value) {
+										setErrorMessage(
+											Liferay.Language.get(
+												'this-field-is-required'
+											)
+										);
+									}
```

Comment:
Aren't we missing a required check here? In addition, we can simplify error message setting, all to something like

```js
if (required) {
	setErrorMessage(
		event.target.value
			? ''
			: Liferay.Language.get(
					'this-field-is-required'
				)
	);
}
```

Reply:
Ok, thanks!!